pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Summary
+
+<!-- Required section. Describe what this PR will do if it is merged. -->
+
+## Details
+
+<!-- Optional section. Give more details if it is sensible to do so. -->
+
+## Tests
```

Comment:
not sure what you are expecting folks to write in "Tests", could you elaborate? seems like is handled as CI?